### PR TITLE
chore: remove c flag & TOKEN_BOT_CLONE from main file

### PR DESCRIPTION
**Pull request by AfaqShuaib09**
Description:
-----------------
This PR removes `-c` flag from the `main.py` file as it is deprecated now.

### Issue:
https://github.com/Luxembourg-Open-Source-Club/BICS-BOT/issues/15

### CODE REVIEW

**Comment by AfaqShuaib09** [comment]
Hey @umutucak & @PedroS235, could you guys please review this PR. I have created it for the issue mentioned here: https://github.com/Luxembourg-Open-Source-Club/BICS-BOT/issues/15. Thank you!

File: main.py [hunk 1]
```diff
@@ -56,17 +56,12 @@ def main(args: vars):
     )
 
     load_extensions(bot)
-
-    if args["clone"]:
-        bot.run(os.getenv("TOKEN_BOT_CLONE"))
-    else:
-        bot.run(os.getenv("TOKEN_BOT"))
+    bot.run(os.getenv("TOKEN_BOT"))
```
Comment (umutucak) [inline main.py:59]: this is good 👍

File: main.py [hunk 2]
```diff
@@ -56,17 +56,12 @@ def main(args: vars):
...
 
 
 if __name__ == "__main__":
     ap = argparse.ArgumentParser()
     ap.add_argument(
-        "-c",
```
Comment (umutucak) [inline main.py:69]: This is also correct, but incomplete. Could we remove the `--clone` argument as well, as the two are the same?

**Review by umutucak — CHANGES_REQUESTED** [review]
Looking pretty good! But not quite complete... Remove `--clone` as well, and then we are ready to merge! Great job 👍

File: main.py (hunk 2 above)
Comment (AfaqShuaib09) [reply main.py:69]: Sure. I will update it in a bit

Comment (umutucak) [reply main.py:69]: Actually, please remove the whole `ap.add_argument(...) block. Not just the `--clone` line, but the entire code block regarding to the argument, starting from Line 64 to 69

Comment (AfaqShuaib09) [reply main.py:69]: Got it

Comment (AfaqShuaib09) [reply main.py:69]: Hey @umutucak & @PedroS235, i've updated the changes. Can you have look now.
Plus can I add a docstring to `main` function?

**Review by PedroS235 — APPROVED** [review]
Very good! Thank you for your contribution.

File: main.py (hunk 2 above)
Comment (PedroS235) [reply main.py:69]: You can add the docstring. Good idea.

**Review by umutucak — APPROVED** [review]
Onwards we go!

**Comment by umutucak** [comment]
@AfaqShuaib09 i forgot about the docstring sorry lol... i am new to this... :) could you make a new PR for that please?

**Comment by AfaqShuaib09** [comment]
> @AfaqShuaib09 i forgot about the docstring sorry lol... i am new to this... :) could you make a new PR for that please?

Sure I will make a PR for this in a bit

**Pull request merged**